.circleci/bash_env.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export GIT_COMMIT=$(git rev-parse --short HEAD)
+export GIT_DIRTY=$(test -n "`git status --porcelain`" && echo "+CHANGES" || true)
+export GIT_DESCRIBE=$(git describe --tags --always --match "v*")
+export GIT_IMPORT=github.com/hashicorp/waypoint/internal/version
```

Comment:
Not sure if we should copy this over at all from waypoint or just remove this script completely :smile: mostly just hoping circle can run these unit tests per pull request so maybe this isn't required here.

Reply:
I agree we don't need all of this